### PR TITLE
[5][6] Footer component and base Landing page

### LINKED ISSUE #5: Add a Footer
As team, we need to create the footer that is going to be at the bottom of each view.

Image

Acceptance criteria:

- Footer design should be similar to figmas
- Buttons can have no onclick events for now

### LINKED ISSUE #6: Create site's base page
As team, we need to create a base page that is gonna be used by all future pages. This page should contain both header and footer from #5 and #4

### PR DESCRIPTION
**Pull request by listerineh**
- Upload svg icons for `SocialMedia` component.
- Add a `Footer` component.
- Add a base `Landing` page with `Header` and `Footer` components.

Preview: https://club-site-oc1zoho5p-danjar27.vercel.app/

Closes #5 
Closes #6

### CODE REVIEW

File: src/components/atoms/SocialMedia.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import LinkedinIcon from '../.././assets/icons/linkedin-icon.svg';
+import TelegramIcon from '../.././assets/icons/telegram-icon.svg';
+import WhatsAppIcon from '../.././assets/icons/whatsapp-icon.svg';
+
+const SocialMedia = () => {
+  return (
+    <div className="flex flex-row gap-x-5">
```
Comment (Danjar27) [inline src/components/atoms/SocialMedia.tsx]: let's extract this <link><imgage></link> logic into its own component, call it socialIcon

File: src/components/molecules/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import SocialMedia from '../atoms/SocialMedia';
+
+const Footer = () => {
+  return (
+    <footer className="flex flex-col absolute w-full justify-center items-center gap-4 bottom-0 px-3 py-14 bg-green">
```
Comment (Danjar27) [inline src/components/molecules/Footer.tsx]: please, simplify the styles that are being used. Many of them are not necessary

File: src/pages/Landing.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Footer from '../components/molecules/Footer';
+import Header from '../components/molecules/Header';
+
+const Landing = () => {
```
Comment (Danjar27) [inline src/pages/Landing.tsx]: We need to pass the children prop to the base page and render it between the header and the footer. Something like

```
({children}) => {
    <header/>
    {children}
    <footer/>
```

File: src/pages/Landing.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import Footer from '../components/molecules/Footer';
```
Comment (Danjar27) [inline src/pages/Landing.tsx]: rename this component to basePage and move it to src/components/templates, please

**Review by Danjar27 — CHANGES_REQUESTED** [review]

File: src/components/molecules/Footer.tsx (hunk 2 above)
Comment (listerineh) [reply src/components/molecules/Footer.tsx]: I dont know if `bottom-0` is necessary, but I think is

File: src/pages/Landing.tsx (hunk 3 above)
Comment (listerineh) [reply src/pages/Landing.tsx]: Done 👍

File: src/components/atoms/SocialMedia.tsx (hunk 1 above)
Comment (listerineh) [reply src/components/atoms/SocialMedia.tsx]: Done 👍

File: src/pages/Landing.tsx (hunk 4 above)
Comment (listerineh) [reply src/pages/Landing.tsx]: Done +1

**Pull request merged**